src/WKWebKit/Defs.cs

```diff
@@ -113,4 +113,28 @@ public enum WKAudiovisualMediaTypes : ulong	{
 		Video = 1 << 1,
 		All = UInt64.MaxValue
 	}
+
+	[Mac (11,3), iOS (14,5), MacCatalyst(14,5)]
```

Comment:
minor: missing space before `(`

Reply:
There are also other MacCatalyst headers in this PR that are missing the same space!